src/pages/home/current-sdk-versions.md

```diff
@@ -66,6 +66,10 @@ The Android BOM (Bill of Materials) artifact has been released to Maven Central.
 
 **Migrate to latest Mobile SDKs for iOS**<br/>If you are currently using older SDK versions, please see the [Migrating to latest Mobile SDKs for iOS](../resources/migration/ios/index.md) guide for next steps.
 
+<InlineAlert variant="info" slots="text"/>
+
+Starting May 7th, 2024, our iOS releases will include signed XCFrameworks. Additionally, we've updated all our 5.0.0 releases with signed XCFrameworks as well.
```

Comment:
A bit redundant to say "Additionally... as well."

I'd just get rid of one of them. (I do this all the time when I speak, so I get the issue. :P )

Reply:
good call out. Have fixed that in all of them.